lib/key/check.js

```diff
@@ -15,7 +15,7 @@ export function keyCheck(info, email, expectEncrypted) {
             throw new Error('Missing or too many UserID packets');
         }
 
-        if (!new RegExp('<' + email + '>$').test(info.user.userId)) {
+        if ((info.user.userId + '').endsWith('<' + email + '>')) {
```

Comment:
Thanks :blush: A few tests failed, one of them is because of this:
```suggestion
        if (!(info.user.userId + '').endsWith('<' + email + '>')) {
```
The other test failures are unrelated and should be gone after merging #91.